README.md

```diff
@@ -92,12 +95,16 @@ except for `libxmlsec1`, which you can install with Homebrew:
 
 ### Running Tests
 
+Make sure you have [PhantomJS](http://phantomjs.org) installed prior to running tests. Install with the following:
+
+    $ brew install phantomjs
```

Comment:
Maybe its enough to just mention this up in the dependencies section. 

it should to failure clear from the failure messages when phantomjs is missing right?